### DISCUSSION TITLE
LXC script for Xteve?

### DISCUSSION THREAD

**Discussion by chenks** [tteck/Proxmox#3120] [Ideas]
hi

any chance there could be a script created for https://github.com/dnsforge-repo/xteve ?
i see one for Threadfin has recently been added, but i prefer the dnsforge build of xteve that has guide2go included.

**Comment by chenks** [comment]
hi @tteck do you think this would be possible?

**Comment by djvetbo** [comment]
Bonjour,
Voici
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/threadfin.sh)"

**Reply by chenks** [reply]
threadfin is not xteve.
also threadfin is a dead product (as per the dev) and threadfin is actually worse than xteve.
